terraform/terraform.tf

```diff
@@ -4,6 +4,7 @@ terraform {
     encrypt = true
     key     = "targets/terraform"
     region  = "eu-west-1"
+    profile = "nixos-prod"
```

Comment:
I'm not that knowledgeable about tf profiles, but it seems weird to me that this is in the `nixos-prod` profile, but in `eu-west-1`, while above the `nixos-prod` profile is assigned to `us-east-1`... Maybe it doesn't matter though.